php-frontend/src/main/java/org/sonar/php/parser/NewPHPGrammar.java

```diff
@@ -19,9 +19,21 @@
  */
 package org.sonar.php.parser;
 
-import com.sonar.sslr.api.typed.GrammarBuilder;
+import org.sonar.php.api.PHPKeyword;
+import org.sonar.php.api.PHPPunctuator;
+import org.sonar.php.tree.impl.expression.ParenthesizedExpressionTreeImpl;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.php.tree.impl.expression.ParenthesizedExpressionTreeImpl'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)